<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 253, 240);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushBu_orig">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>изначальное изображение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_bd">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>БД</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>101</width>
     <height>154</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="povorot">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>12</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>поворот на</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pravod">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);</string>
      </property>
      <property name="text">
       <string>90 вправо</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="levod">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);</string>
      </property>
      <property name="text">
       <string>90 влево</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pravos">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);</string>
      </property>
      <property name="text">
       <string>180</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="zerc">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>отзеркалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="izobr">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>81</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="blue">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(41, 80, 255);
font: 75 8pt &quot;Segoe Print&quot;;
</string>
      </property>
      <property name="text">
       <string>синий</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_red">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 6, 10);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>красный</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="green">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(47, 255, 0);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>зеленый</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>371</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="yarkost">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>Яркость</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="comBY_yvel">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>увеличеть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="contrast">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>контраст</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="contr_yvelich">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>увеличить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>81</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="hb">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>серый</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="negative">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(12, 1, 0);
color: rgb(255, 255, 255);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>негатив</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_H2O">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(149, 206, 255);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>H2O</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="razmutie">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(219, 255, 252);
font: 75 8pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>размытие</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prozrachnost">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>прозрачность</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 75 11pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>эффекты гравюры</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="but_contur">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>контур</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="but_contur1">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>контур Pro</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="but_contur2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>тиснение</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="but_contur3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 184);
font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>чб</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="sepeia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 153, 85);
font: 75 8pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>сепия</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 253, 240);

font: 75 11pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>Резкость</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="recz_yvel">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>увеличить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="but_lychi">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>410</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 244, 124);
background-color: rgb(255, 252, 170);
font: 75 8pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>лучи рассвета</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 7pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>Ctrl + S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_snow">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 254, 255);
font: 75 8pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>снег</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_night">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
font: 75 8pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>сумерки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_summer">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Segoe Print&quot;;
background-color: rgb(255, 203, 120);</string>
   </property>
   <property name="text">
    <string>лето</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_gold">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
font: 75 8pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>Золото</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_dreems">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 120, 104);
font: 75 8pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>сны</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
